docs/source/installation.rst

```diff
@@ -37,7 +42,7 @@ For a standard installation with pip_, execute:
 
 .. code:: bash
 
-    $ pip install signac signac-flow
+    $ pip install synced_collections signac signac-flow
```

Comment:
As discussed above, I think I’d keep the previous command for simplicity. Users never need to interact directly with synced_collections.